### PR TITLE
add oct 2016 devtools digest

### PR DESCRIPTION
**Pull request by kaycebasques**
I'll leave this open for a bit in case anyone on DevTools team wants to review...

### CODE REVIEW

File: src/content/en/updates/2016/10/devtools-digest.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: New Console features, updates on the context selector bug, and the new UC Browser user agent.
+
+{# wf_updated_on: 2016-10-18 #}
+{# wf_published_on: 2016-10-18 #}
+{# wf_tags: devtools #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
```
Comment (jpmedley) [inline src/content/en/updates/2016/10/devtools-digest.md]: For updates/ you also need a {# wf_featured_snippet: #} tag.

File: src/content/en/updates/2016/10/devtools-digest.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: New Console features, updates on the context selector bug, and the new UC Browser user agent.
+
+{# wf_updated_on: 2016-10-18 #}
+{# wf_published_on: 2016-10-18 #}
+{# wf_tags: devtools #}
```
Comment (jpmedley) [inline src/content/en/updates/2016/10/devtools-digest.md]: Needs a chrome56 tag.

File: src/content/en/updates/2016/10/devtools-digest.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+Hey hey, [Kayce](https://twitter.com/kaycebasques) here, back again for
+another digest to update you on what's new in DevTools since last month.
+
+[Crbug]: https://bugs.chromium.org/p/chromium/issues/list?can=1&q=component%3APlatform%3EDevTools+-status%3AUnconfirmed+-status%3AUntriaged++-status%3AAvailable+-status%3AAssigned+-status%3AStarted+-status%3AExternalDependency+-status%3ADuplicate+-status%3AWontFix+-status%3AArchived+&sort=-modified&colspec=ID+Pri+M+Stars+ReleaseBlock+Component+Status+Owner+Summary+OS+Modified&x=m&y=releaseblock&cells=ids
+
+## New features in the Console {: #console }
+
+In Chrome 56, which is currently Canary, the DevTools Console is powered by
```
Comment (jpmedley) [inline src/content/en/updates/2016/10/devtools-digest.md]: in Canary

File: src/content/en/updates/2016/10/devtools-digest.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+[CodeMirror](https://codemirror.net/). This enables a whole bunch of new
+features, like:
+
+**Syntax highlighting as you type**. Previously, DevTools could only
+highlight syntax after a code block had been evaluated.
+
+**Matching bracket highlighting**. Because no one got time for those annoying
```
Comment (jpmedley) [inline src/content/en/updates/2016/10/devtools-digest.md]: grammar

File: src/content/en/updates/2016/10/devtools-digest.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+`Uncaught SyntaxError: Unexpected token {` errors.
+
+**Smart return**. When you type out a multi-line code block, DevTools
+now knows whether to create a new line or execute your code each time you
+press <kbd>Enter</kbd>. You can still force DevTools to create a new line by
+holding <kbd>Shift</kbd> before you press <kbd>Enter</kbd>.
```
Comment (jpmedley) [inline src/content/en/updates/2016/10/devtools-digest.md]: Are you saying I no longer need to write functions on a single line? Might make that clearer. You might compare this to command line interpreters in other languages like Python for instance. (I think Ruby might also do that. Ask around.)

File: src/content/en/updates/2016/10/devtools-digest.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+![non-top context highlighted
+red](/web/updates/images/2016/10/non-top-context.png)
+
+## New user agent: UC Browser
+
+You can now select UC Browser for iOS, Android, or Windows Phone from
+the **Network conditions** [drawer
+tab](/web/tools/chrome-devtools/settings#drawer-tabs).
+
```
Comment (jpmedley) [inline src/content/en/updates/2016/10/devtools-digest.md]: 'drawer tab'? Pick one or the other.

File: src/content/en/updates/2016/10/devtools-digest.md (hunk 1 above)
Comment (petele) [inline src/content/en/updates/2016/10/devtools-digest.md]: I **strongly** recommend the `wf_featured_snippet` tag, it's what's used for the description on the index page, vs the `description:` which is used as the `<meta description`. You can also include HTML in the `wf_featured_snippet` to be a little more attention grabbing.

File: src/content/en/updates/2016/10/devtools-digest.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+![UC Browser user agent](/web/updates/images/2016/10/uc-browser.png)
+
+## Sharing is caring
+
+As always, we'd love to hear your feedback or ideas on anything DevTools
+related.
+
+* Ping us at [ChromeDevTools](https://twitter.com/chromedevtools) on Twitter
+  for brief questions or feedback, or to share new ideas.
+* For longer discussions, the [mailing list](https://groups.google.com/forum/#!forum/google-chrome-developer-tools/topics) or [Stack Overflow](http://stackoverflow.com/questions/tagged/google-chrome-devtools) are your best bets.
```
Comment (petele) [inline src/content/en/updates/2016/10/devtools-digest.md]: Nit: If both http and https domains are supported by an outbound link, preference for `//stackoverflow.com/....`

Comment (kaycebasques) [inline src/content/en/updates/2016/10/devtools-digest.md]: Shouldn't we be encouraging HTTPS everywhere, though?

**Comment by kaycebasques** [comment]
Don't think the DevTools team will be reviewing this. Updated per feedback I've already received. This is good to go unless there's any more feedback.

**Comment by marccantin** [comment]
Re: STOP YOUR FUCKY SPAM!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

2016-10-21 12:46 GMT-04:00 Joe Medley <notifications@github.com>:

**Pull request merged**